doc/en/user/source/production/config.rst

```diff
@@ -3,121 +3,196 @@
 
-You can also use a more generic non-spatial caching system, such as `OSCache <http://www.opensymphony.com/oscache/>`__ (an embedded cache service) or `Squid <http://www.squid-cache.org>`__ (a web cache proxy).
+1. Create a role to be used for data administration.
 
-Caching is also possible for WFS layers, in a very limited fashion. For DataStores that don't have a quick way to determine feature counts (e.g. shapefiles), enabling caching can prevent querying a store twice during a single request. To enable caching, set the Java system property ``org.geoserver.wfs.getfeature.cachelimit`` to a positive integer. Any data sets that are smaller than the cache limit will be cached for the duration of a request, which will prevent the dataset from being queried a second time for the feature count. Note that this may adversely affect some types of DataStores, as it bypasses any feature count optimizations that may exist.
+2. Optional: Assign this role ot to user to associate GROUP_ADMIN.
+   
+   This built-in role provides access to User / Group/ Role information. 
 
-Welcome page selectors
-----------------------
+The combination of admin permission for a workspace and GROUP_ADMIN access provides a good combination for a data administrator to mange team access, and provide access to a workspace.
 
+GeoServer Administrator Guidance
+--------------------------------
 
-The workspace and layer selectors might take a lot of time to fill up against large catalogs. Because of this, GeoServer tries to limit the time taken to fill them (by default, 5 seconds), and the number of items in them (by default, ``1000``), and will fall back on simple text fields if the time limit is reached.
+The GeoServer administration console is optimized to support operation by a trusted GeoServer Administrator, who is often the same System Administrator.
 
-In some situations, that won't be enough and the page might get stuck anyways. The following properties can be used to tweak the behavior:
+GeoServer provides extensive control using the Administration Console of global settings allowing an administrator to adapt the application to the operational enviornment:
 
-*  ``GeoServerHomePage.selectionMode`` : can be set to ``text`` to always use simple text fields, ``dropdown`` to always use dropdowns, or ``auto`` to use the default automatic behavior.
-* ``GeoServerHomePage.selectionTimeout`` : the time limit in milliseconds, defaults to ``5000``.
-* ``GeoServerHomePage.selectionMaxItems`` : the maximum number of items to show in the dropdowns, defaults to ``1000``.
+* :ref:`proxy_base`
+* :ref:`config_globalsettings_log_location`
 
-When using ``text`` selection mode the page description is static, no longer offering of available workspace and layers.
+Management of a web service using an administration console is a common practice and expectation when running GeoServer as a windows web service.
```

Comment:
expected?

Reply:
I am not sure how to explains since it is such a foreign idea for Linux admins